doc/alitv-filter.md

```diff
@@ -0,0 +1,69 @@
+# NAME
+
+AliTV filter - filters AliTV JSON files for postprocessing
+
+# SYNOPSIS
+
+    # Filter input file based on settings in JSON
+    alitv-filter.pl --input in.json --output out.json
+
+    # Filter input file based on settings in JSON in combination with
+    # the command line settings
+    alitv-filter.pl --input in.json --output out2.json --min-link-length 10000 --min-link-identity 90%
+
+    # Filter input file based the command line settings and ignoring
+    # JSON settings
+    alitv-filter.pl --input in.json --output out3.json --ignore-json --min-link-length 10000 --min-link-identity 90%
+
+# DESCRIPTION
+
+The script filters AliTV JSON files for postprocessing.
+
+# OPTIONS
+
+- `--help|?|man`  Help
+
+    Shows description of this programs, its usage, and its parameters/options.
+
+- --input  Input file
+
+    The name of the input file. Default value is input from STDIN which
+    might be switched on by `--input -`.
```

Comment:
Rephrased: "Default value is input from STDIN which can be manually enabled by --input -"